pyxem/signals/strain_map.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+from hyperspy.signals import Signal2D
+import numpy as np
+from pyxem.signals import push_metadata_through
+
+def _get_rotation_matrix(x_new):
+        """Internal function to get the rotation matrix that takes [1,0] to x_new
+
+        Parameters
+        ----------
+        x_new : list
+            The coordinates of a point that lies on the new 'x' axis
+        Returns
+        -------
+        R : 2 x 2 numpy asarray
+            Contains the correct rotation matrix
+        """
+        # TODO: facilitate rotations where the original basis is something
```

Comment:
This can be handled by using a transpose on the rotation that takes x_current to [1,0], so no extra functionality needed here.